specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/examples/RedisCacheForceReboot.json

```diff
@@ -6,7 +6,11 @@
     "subscriptionId": "subid",
     "parameters": {
       "shardId": 0,
-      "rebootType": "AllNodes"
+      "rebootType": "AllNodes",
```

Comment:
@Priya91 Regarding the example here, I didn't think we really intended for people to send both Ports and RebootType in the same request. Can we make the example just show one or the other? Or maybe even do two examples if we want to show both? (Reboot by ports, reboot by type / shards?)